File: CS/WindowsApplication3/Custom Ribbon/MyRibbonPanelGroupPainter.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;
using DevExpress.Skins;
using DevExpress.XtraBars.Ribbon.ViewInfo;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Ribbon.Drawing;

namespace DXSample {

    public class MyRibbonPanelGroupPainter : RibbonPanelGroupPainter
    {
        public MyRibbonPanelGroupPainter() : base() { }
        protected override void DrawItem(RibbonViewInfo viewInfo, DevExpress.Utils.Drawing.GraphicsCache cache, RibbonItemViewInfo itemInfo)
        {
            MyRibbonItemViewInfoCalculator.DrawItem(cache, itemInfo);
        }

    }
}

File: VB/WindowsApplication3/Custom Ribbon/MyRibbonPanelGroupPainter.vb
Imports DevExpress.XtraBars.Ribbon.ViewInfo
Imports DevExpress.XtraBars.Ribbon.Drawing

Namespace DXSample

    Public Class MyRibbonPanelGroupPainter
        Inherits RibbonPanelGroupPainter

        Public Sub New()
            MyBase.New()
        End Sub

        Protected Overrides Sub DrawItem(ByVal viewInfo As RibbonViewInfo, ByVal cache As DevExpress.Utils.Drawing.GraphicsCache, ByVal itemInfo As RibbonItemViewInfo)
            MyRibbonItemViewInfoCalculator.DrawItem(cache, itemInfo)
        End Sub
    End Class
End Namespace
